### PR TITLE
Add runbook and update codeowners

### PR DESCRIPTION
**Pull request by bboykov**

### CODE REVIEW

File: runbooks/runbook.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+FullyAutomated
+
+## Failback Process Type
+
+FullyAutomated
+
+## Failover Details
+
+The service is deployed in both Delivery clusters. The failover guide for the cluster is located here: <https://github.com/Financial-Times/upp-docs/tree/master/failover-guides/delivery-cluster>
```
Comment (ekaneva) [inline runbooks/runbook.md:68]: Those are not correct, Publish history is not deployed on the Delivery nor any of our other clusters.
Maybe @georgi-denchew could be able to explain better how to change these sections.

**Review by ekaneva — CHANGES_REQUESTED** [review]

File: runbooks/runbook.md (hunk 1 above)
Comment (ekaneva) [inline runbooks/runbook.md:68]: Publish Carousel is deployed in the Publishing clusters, so please update Architecture Type and Failover docs accordingly.

File: runbooks/runbook.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+## Key Management Details
+
+There is no key rotation procedure for this system.
+
+## Monitoring
+
+Look for the pods in the cluster health endpoint and click to see pod health and checks:
+
+- <https://upp-prod-delivery-eu.upp.ft.com/__health/>
+- <https://upp-prod-delivery-us.upp.ft.com/__health/>
```
Comment (ekaneva) [inline runbooks/runbook.md:103]: Publishing cluster.

**Review by ekaneva — CHANGES_REQUESTED** [review]

File: runbooks/runbook.md (hunk 1 above)
Comment (bboykov) [reply runbooks/runbook.md]: 👍

File: runbooks/runbook.md (hunk 2 above)
Comment (bboykov) [reply runbooks/runbook.md]: 👍

File: runbooks/runbook.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+FullyAutomated
+
+## Failback Process Type
+
+FullyAutomated
+
+## Failover Details
+
+The service is deployed in both clusters. The failover guide for the cluster is located here: <https://github.com/Financial-Times/upp-docs/tree/master/failover-guides/publishing-cluster>
```
Comment (ekaneva) [inline runbooks/runbook.md:68]:
```suggestion
The service is deployed in both Publishing clusters. The failover guide for the cluster is located here: <https://github.com/Financial-Times/upp-docs/tree/master/failover-guides/publishing-cluster>
```

File: runbooks/runbook.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+No
+
+## Contains Sensitive Data
+
+No
+
+## Failover Architecture Type
+
+ActiveActive
```
Comment (ekaneva) [inline runbooks/runbook.md:56]:
```suggestion
ActivePassive
```

**Review by ekaneva — CHANGES_REQUESTED** [review]

**Review by ekaneva — DISMISSED** [review]

File: runbooks/runbook.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+## Key Management Details
+
+There is no key rotation procedure for this system.
+
+## Monitoring
+
+Look for the pods in the cluster health endpoint and click to see pod health and checks:
+
+- <https://upp-prod-publish-eu.upp.ft.com/__health/>
```
Comment (MiroslavGatsanoga) [inline runbooks/runbook.md:102]: Those should be the service health endpoints:
https://upp-prod-publish-eu.upp.ft.com/__health/__pods-health?service-name=publish-carousel
https://upp-prod-publish-us.upp.ft.com/__health/__pods-health?service-name=publish-carousel

**Review by MiroslavGatsanoga — DISMISSED** [review]

File: runbooks/runbook.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+NotApplicable
+
+## Key Management Details
+
+There is no key rotation procedure for this system.
+
+## Monitoring
+
+Look for the pods in the cluster health endpoint and click to see pod health and checks:
+
```
Comment (ekaneva) [inline runbooks/runbook.md:101]: In that case, this sentence is redundant:
```suggestion
```

**Review by ekaneva — CHANGES_REQUESTED** [review]

**Review by ekaneva — APPROVED** [review]
Looks good to me, just squash the commits once you're done.

**Review by MiroslavGatsanoga — APPROVED** [review]

**Pull request merged**